app/cdap/components/GridTable/components/Breadcrumb/index.tsx

```diff
@@ -19,29 +19,32 @@ import Breadcrumbs from '@material-ui/core/Breadcrumbs';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { IBreadcrumbProps } from './types';
+import T from 'i18n-react';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import { useStyles } from './styles';
 
-export default function BreadCrumb({ datasetName }) {
+export default function({ breadcrumbsList }: IBreadcrumbProps) {
```

Comment:
Why does this not use the `Breadcrumb` component?

Reply:
Updated the code.